### PR TITLE
Profile conformance mk

### PR DESCRIPTION
**Pull request by markkramerus**
Added individual conformance statements to each profile.

### CODE REVIEW

File: input/fsh/CancerCondition.fsh [hunk 1]
```diff
@@ -31,7 +31,10 @@ Description: "Records the history of the primary cancer condition, the original
...
 
-Conformance statement: All Conditions associated with an mCODE patient with a code from value set PrimaryOrUncertainBehaviorCancerDisorderVS (or a code from any other terminology system with equivalent semantics) SHALL conform to this profile."
+Conformance statement:
+
+Condition resources associated with an mCODE patient with a Condition.code in the value set PrimaryOrUncertainBehaviorCancerDisorderVS MUST conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patient that would reasonably be expected to conform to this profile SHOULD be published in this form, for example, when employing a code that extends the PrimaryOrUncertainBehaviorCancerDisorderVS value set. Any resource intended to conform to this profile SHOULD populate meta.profile accordingly."
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/CancerCondition.fsh]: I did it this way: [mCODE patient](conformance.html#mCODE-Patients)

File: input/fsh/CancerDiseaseStatus.fsh [hunk 2]
```diff
@@ -4,9 +4,12 @@ Id:       mcode-cancer-disease-status
...
+Note: The LOINC code chosen to represent this observation (LOINC 88040-1, Response to cancer treatment) does not precisely match the meaning of this profile, but it is the closest available LOINC code at the present time. It is acknowledged that the disease status is different than the status of the disease due to treatment, although in the context of an oncologist visit, disease status can mean response to treatment for patients under their care. However, the LOINC code 88041-2 is more granular than the definition of the profile because cancer disease status is observable regardless of whether the patient is under treatment. The plan is to request a new LOINC code that represents cancer disease status, as it is defined here, and replace the current LOINC code with the new code before normative publication of mCODE.
+
+Conformance statement: 
+
+Observation resources associated with an mCODE patient with Observation.code LOINC 88040-1 MUST conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patient that would reasonably be expected to conform to this profile SHOULD be published in this form."
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/CancerDiseaseStatus.fsh:11]: It is intentional that some resources don't have this requirement, generally those that are least mCODE-specific.

File: input/fsh/CancerPatient.fsh [hunk 3]
```diff
@@ -2,7 +2,12 @@ Profile: CancerPatient
 Parent: USCorePatient
 Id: mcode-cancer-patient
 Title: "Cancer Patient"
-Description: "A patient who has been diagnosed with or is receiving medical treatment for a malignant growth or tumour."
+Description: "A patient who has been diagnosed with or is receiving medical treatment for a malignant growth or tumour.
+
+Conformance Statement:
+
+Patient resources representing mCODE patients (as defined in [Conformance](conformance.html#mCODE-Patients)) MUST conform to this profile. Any resource intended to conform to this profile MAY populate meta.profile accordingly."
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/CancerPatient.fsh:9]: OK, I was on the fence about this anyway.

File: input/fsh/CancerRelatedMedicationStatement.fsh [hunk 4]
```diff
@@ -2,7 +2,11 @@ Profile:  CancerRelatedMedicationStatement
 Parent:   MedicationStatement
 Id:       mcode-cancer-related-medication-statement
 Title:    "Cancer-Related Medication Statement"
-Description:    "A record of the use of a medication (individual administration or entire course) for a condition that is related to a primary or secondary cancer condition. The use may be reported by the patient or clinician and administration does not have to be directly observed."
+Description:    "A record of the use of a medication (individual administration or entire course) for a condition that is related to a primary or secondary cancer condition. The use may be reported by the patient or clinician and administration does not have to be directly observed.
+
+Conformance statement:
+
+MedicationStatement resources associated with an mCODE patient SHOULD conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patient that would reasonably be expected to conform to this profile SHOULD be published in this form."
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/CancerRelatedMedicationStatement.fsh:9]: Medication statement is the subject of a separate PR. I would rather discuss it there.

File: input/fsh/CancerRelatedMedicationStatement.fsh [hunk 5]
```diff
@@ -16,8 +20,6 @@ Description:    "A record of the use of a medication (individual administration
 * extension contains
     TreatmentIntent named treatmentIntent 0..1 MS and
     TerminationReason named terminationReason 0..* MS
-//* reasonReference only Reference(PrimaryCancerCondition | SecondaryCancerCondition | Observation | DiagnosticReport | ComorbidConditionParent)
 * effective[x] 1..1  // change from 0..1 to 1..1
-* reasonCode 0..1
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/CancerRelatedMedicationStatement.fsh:21]: See my other PR. Resolved there.

File: input/fsh/ComorbidCondition.fsh [hunk 6]
```diff
@@ -283,7 +283,7 @@ Profile: ElixhauserWeightLossComorbidity
 Parent:  ComorbidConditionParent
 Id: mcode-elixhauser-weight-loss-comorbidity
 Title: "Elixhauser Weight Loss Comorbidity"
-Description:  "Elixhauser comorbid condition profile for Weight Loss."
+Description:  "Elixhauser comorbid condition profile for Weight Loss. This profile is provided primarily so its canonical URL can be used in meta.profile to indicate a Condition instance falls into this Elixhauser comorbidity category."
 * ^abstract = false
 * code from ElixhauserWeightLossVS (extensible)
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/ComorbidCondition.fsh:291]: I'm not sure what you are suggesting.

File: input/fsh/ComorbidCondition.fsh [hunk 7]
```diff
@@ -166,7 +166,7 @@ Profile: ElixhauserLymphomaComorbidity
 Parent:  ComorbidConditionParent
 Id: mcode-elixhauser-lymphoma-comorbidity
 Title: "Elixhauser Lymphoma Comorbidity"
-Description:  "Elixhauser comorbid condition profile for Lymphoma."
+Description:  "Elixhauser comorbid condition profile for Lymphoma. This profile is provided primarily so its canonical URL can be used in meta.profile to indicate a Condition instance falls into this Elixhauser comorbidity category."
 * ^abstract = false
 * code from ElixhauserLymphomaVS (extensible)
 */
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/ComorbidCondition.fsh:174]: The ComorbidConditionParent profile provides a base profile for specific comorbidity profiles. In mCODE, one profile is defined for each comorbid condition in the Elixhauser Comorbidity Index, **with the exception of cancer conditions (the index disease in the context of mCODE).** These profiles allow implementers to tag relevant resources as conforming to their respective profiles using [`meta.profile`](https://www.hl7.org/fhir/resource-definitions.html#Meta.profile). Value sets are also defined for each comorbid condition in the Index.

**Review by masnick — CHANGES_REQUESTED** [review]
Just a few typo fixes, and I added to the radiation procedure statement so it parallels the surgical procedure statement.

File: input/fsh/ComorbidCondition.fsh [hunk 8]
```diff
@@ -14,7 +14,7 @@ Profile: ElixhauserHIVAIDSComorbidity
 Parent:  ComorbidConditionParent
 Id: mcode-elixhauser-hiv-aids-comorbidity
 Title: "Elixhauser HIV AIDS Comorbidity"
-Description:  "Elixhauser comorbid condition profile for HIV-AIDS."
+Description:  "Elixhauser comorbid condition profile for HIV-AIDS. This profile is provided primarily so its canonical URL can be used in meta.profile to indicate a Condition instance falls into this Elixhauser comorbidity category."
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/ComorbidCondition.fsh:19]: Fixed

File: input/fsh/ComorbidCondition.fsh (hunk 6 above)
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/ComorbidCondition.fsh:291]: I am not suggesting the list of comorbidities be saved.

File: input/fsh/CancerCondition.fsh [hunk 9]
```diff
@@ -33,7 +33,7 @@ Cancer staging information summarized in this profile should reflect the most re
...
+Condition resources associated with an [mCODE patient](conformance.html#mCODE-Patients)with a Condition.code in the value set PrimaryOrUncertainBehaviorCancerDisorderVS MUST conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patientthat would reasonably be expected to conform to this profile SHOULD be published in this form, for example, when employing a code that extends the [PrimaryOrUncertainBehaviorCancerDisorderVS](ValueSet-mcode-primary-or-uncertain-behavior-cancer-disorder-vs.html) value set. Any resource intended to conform to this profile SHOULD populate meta.profile accordingly."
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/CancerCondition.fsh:36]: fixed

File: input/fsh/CancerRelatedProcedures.fsh [hunk 10]
```diff
@@ -47,8 +47,7 @@ Description: "A surgical action addressing a cancer condition. The scope of this
 
 Conformance statement:
 
-Procedure resources associated with an mCODE patient with Procedure.category SNOMED-CT 387713003 MAY conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patient that would reasonably be expected to conform to this profile SHOULD be published in this form."
-
+Procedure resources associated with an [mCODE patient](conformance.html#mCODE-Patients) with Procedure.category SNOMED-CT 387713003 MAY conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patientthat would reasonably be expected to conform to this profile SHOULD be published in this form.  Specifically, we expect that any surgical procedure related to the treatment of a `PrimaryCancerCondition` or `SecondaryCancerCondition` would be published in this form."
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/CancerRelatedProcedures.fsh:50]: fixed

File: input/fsh/CancerRelatedProcedures.fsh [hunk 11]
```diff
@@ -6,7 +6,7 @@ Description: "A radiological treatment addressing a cancer condition. The scope
 
 Conformance statement:
 
-Procedure resources associated with an mCODE patient with Procedure.category SNOMED-CT 53438000 MAY conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patient that would reasonably be expected to conform to this profile SHOULD be published in this form."
+Procedure resources associated with an [mCODE patient](conformance.html#mCODE-Patients) with Procedure.category SNOMED-CT 53438000 MAY conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patientthat would reasonably be expected to conform to this profile SHOULD be published in this form."
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/CancerRelatedProcedures.fsh:9]: Added.

**Pull request merged**